app/models/user.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 class User < ApplicationRecord
+  scope :sorted_by_name, -> { order(name: :asc) }
```

Comment:
```suggestion
  scope :sorted_by_name, -> { order :name }
```
asc là default, em lượt bỏ dc
với scope viết sau validate nha bạn

Reply:
Zạaaaaaaaaaaa